doc/development.md

```diff
@@ -28,10 +28,6 @@ Install additional packages for Fedora, CentOS, and RHEL:
 
     sudo dnf install -y `cat automation/check-patch.packages`
 
-Create virtual environment for vdsm:
-
-    make venv
-
```

Comment:
Why remove this? this belongs to the initial setup that someone must do once, unlike
activating the virtual env that must be done before running the tests.

Reply:
I got the impression that you guys didn't like to mix that in here, because it needs to be done only for running the tests, and not building. Or does it matter, at what time it is executed? I can also leave it in, of course, but I don't think the tests-doc would be complete without mentioning it again, redundantly.